### PR TITLE
Slow down arrows in water

### LINKED ISSUE #5036: Arrows should slow down under water
Client version: 1.12.2
Server OS: Linux
Cuberite Commit id: a0afd132330ab538848d5ac263c021c1247fd688

### Expected behavior
Arrows should slow down underwater, this behaviour was added in the 1.8 update.

### Actual behavior
Arrows fly underwater as if they were shot above water, and don't slow down at all.

### Steps to reproduce the behavior
Shoot an arrow above and underwater and notice no velocity difference.

### PR DESCRIPTION
**Pull request by Persson-dev**
Arrow now slow down in water. Fixes #5036. The value of ARROW_WATER_FRICTION might not be the same as vanilla but it looks good anyway.

### CODE REVIEW

File: src/Entities/ArrowEntity.cpp [hunk 1]
```diff
@@ -194,6 +194,10 @@ void cArrowEntity::Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 			m_IsInGround = false;  // Yes, begin simulating physics again
 		}
 	}
+	else if (IsInWater())  // Arrow in water?
+	{
+		ApplyFriction(m_Speed, ARROW_WATER_FRICTION, static_cast<float>(a_Dt.count()));  // Yes, slow down arrow
+	}
```
Comment (12xx12) [inline src/Entities/ArrowEntity.cpp:200]: Is this just some weird formatting from the web interface?

**Review by 12xx12 — COMMENTED** [review]
Hi and welcome to the project.
Thanks for your contribution and adding yourself to the CONTRIBUTORS file.

I just have one minor comment, and we're ready to go

File: src/Entities/ArrowEntity.cpp (hunk 1 above)
Comment (Persson-dev) [reply src/Entities/ArrowEntity.cpp:200]: I'm sorry but i don't see any problems. It looked fine on vscode and the CheckStyle script said that everything is okay.

**Review by bearbin — APPROVED** [review]

File: src/Entities/ArrowEntity.cpp (hunk 1 above)
Comment (bearbin) [reply src/Entities/ArrowEntity.cpp:200]: I don't see anything wrong here. Were you talking about the line-wrap?

Comment (12xx12) [reply src/Entities/ArrowEntity.cpp:200]: For me the indentation is off. But only in the web interface.

**Pull request merged**